scipy/interpolate/_fitpack_repro.py

```diff
@@ -846,7 +847,7 @@ def make_splrep(x, y, *, w=None, xb=None, xe=None, k=3, s=0, t=None, nest=None):
 
 def make_splprep(x, *, w=None, u=None, ub=None, ue=None, k=3, s=0, t=None, nest=None):
     r"""
-    Find a smoothed B-spline representation of a parametric N-D curve.
+    Create a smoothing parametric B-spline that minimizes derivative jumps s.t. LSQ constraint.
```

Comment:
Please keep the parametric curve here. Otherwise it's too hard to tell this from `make_splrep`

(splrep is for a function, splPrep is for a parametric curve.)

Reply:
Right, I know. I thought "parametric B-spline" would make it clear enough, and I thought it stood out when the two are right next to one another:
<img width="663" alt="image" src="https://github.com/user-attachments/assets/3b029cb4-9453-45f4-afc7-f0f05ef159ef" />

Can you suggest a specific wording? Do you mean change:
> Create a smoothing parametric B-spline that minimizes derivative jumps...

to

> Create a smoothing parametric _curve_ B-spline that minimizes derivative jumps...

?

Originally, I was trying to keep this within the 89 character limit, but I think `noqa:E501` is OK here.

Reply:
I'd call it a parametric B-spline curve

Reply:
```suggestion
    Create a smoothing parametric B-spline curve that minimizes derivative jumps s.t. LSQ constraint.
```

Reply:
Huh, GH web UI does not allow to apply two suggestions to a single line, so I accepted one and recreated the other below.